src/osyris/plot/wrappers.py

```diff
@@ -126,3 +126,50 @@ def scatter(ax, x, y, data, **kwargs):
         return coll
     else:
         return ax.scatter(x, y, **default_args)
+
+def line_integral_convolution(ax, x, y, z, **kwargs):
+    """
+    Wrapper that plots a line integral convolution of a vector field.
+    Uses alpha blending to merge the LIC with a user-requested color.
+    """
+    import lic as lic
+    from matplotlib.cm import ScalarMappable
+    from matplotlib.colors import LogNorm
+
+    #compute line integral convolution
+    try:
+        lic_res = lic.lic(z[...,1], z[...,0], length=kwargs["length"])
+        del kwargs["length"]
+    except KeyError:
+        lic_res = lic.lic(z[...,1], z[...,0], length=30)
+
+    plot_args = {**kwargs}
+    plot_args["extent"] = [np.min(x),np.max(x),np.min(y),np.max(y)]
+    plot_args["origin"] = "lower"
+
+    alpha_blending = True
```

Comment:
If you list `color` in the function args, you should be able to replace `alpha_blending` with `color is not None`